### ISSUE TITLE
Service Fabric Explorer Left Column MaxWidth = 900px?

### ISSUE THREAD

**Issue by jackbond** [Microsoft/service-fabric#294]
I'm really confused by anyone who makes a decision like this. Never mind the fact that I've got a gaming monitor that's 6000 pixels wide. Whoever coded that literally ADDED code that, not only SERVES ABSOLUTELY NO PURPOSE, but is actually detrimental to the user experience. Unfortunately removing it now will require a story, someone putting it in the backlog, reviewing, delaying, etc, etc. Boggles the mind.

**Comment by oanapl** [comment]
@jackbond , thank you for reporting. Service Fabric Explorer is open-sourced [here](https://github.com/Microsoft/service-fabric-explorer), so you can open new features directly there. @jeffj6123 , can you please look into this?

**Comment by jeffj6123** [comment]
Yes, I can look into removing the max width for the left column.
